src/rp2_common/hardware_rcp/include/hardware/rcp.h

```diff
@@ -15,13 +15,15 @@
  */
 
 // ----------------------------------------------------------------------------
-// RCP instructions (this header is Arm-only)
-#if defined(PICO_RP2350) && !defined(__riscv)
+// RCP masks (this header is RP2350 only)
+#if PICO_RP2350
 
 #define RCP_MASK_TRUE   _u(0xa500a500)
 #define RCP_MASK_FALSE  _u(0x00c300c3)
 #define RCP_MASK_INTXOR _u(0x96009600)
 
+// RCP instructions (this header is Arm-only)
```

Comment:
It seems weird that line 18 claims "(this header is RP2350 only)" but line 25 claims "(this header is Arm-only)" :confused: 
Perhaps "these defines" or "this section" or "this code" or similar?

Reply:
Removed the RP2350 only mention, and changed this one to "(these instructions are Arm-only)"